bin/resources/GameIndex.yaml

```diff
@@ -5127,19 +5127,15 @@ SCPS-11018:
   name: "Yoake no Mariko"
   region: "NTSC-J"
 SCPS-11019:
-  name: "Bravo Music - Chou-Meikyokuban [Limited Edition] [Disc1of2]"
+  name: "Bravo Music - Chou-Meikyokuban [Genteiban]"
   region: "NTSC-J"
   memcardFilters:
     - "SCPS-11023"
     - "SCPS-11019"
     - "SCPS-11020"
 SCPS-11020:
-  name: "Bravo Music - Chou-Meikyokuban [Limited Edition] [Disc2of2]"
+  name: "Otostaz [Taikenban]"
```

Comment:
this was already correct <https://psxdatacenter.com/psx2/games2/SCPS-11019.html>

Reply:
No, it wasn't.
The disc 2 is the demo of Otostaz...
https://www.tradergames.fr/206807-large_default/bravo-music-chou-meikyokuban-limited-edition-sony-playstation-2-ps2-ntsc-jpn-occasion.jpg

Reply:
Thanks for the clearer photo of the DVD 😅